packages/core/src/components/spinner/spinner.scss

```diff
@@ -0,0 +1,70 @@
+@use '../../animations';
+@use '../../helpers';
+@use '../../mixins';
+@use '../includes';
+
+@mixin Spinner() {
+  @if not includes.the('spin-animation') {
+    @include animations.Spin();
+  }
+
+  .ods-spinner {
+    animation: ods-spin 0.6s linear infinite;
+    border: 0 solid helpers.color('background-spinner');
+    border-left-color: helpers.color('content-spinner');
```

Comment:
This is real clever, not needing two elements since the filled area is 1/4 of the total.

However this won't work for determinate states.
Should I just discount it as future work?
We won't be able to release this before then, since it'll most likely be a breaking change.

Reply:
This will have to be released next week.

I'm fine with a breaking change.

The "determinate" component is not finished, I don't really know what and if it will even be needed.

TBH it's similar as also this component, sort of developing without having 100% design handover.

What would you suggest then?

Reply:
It'll have to be two elements.

Like my other comment below, I'd recommend using pseudo elements.

```scss
.ods-spinner { // container
  &:before { // spinner background }
  &:after { // spinner fill/content }
}
```

Reply:
Thanks, I did follow your recommendation.

Though I still blended in both "background" and "content" into one border of a pseudo element.